integration/start.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# A helper script to implement restart_container when the docker runtime isn't available.
+#
+# Usage:
+#   Copy start.sh and restart.sh to your container working dir.
+#
+#   Make your container entrypoint:
+#   ./start.sh path-to-binary [args]
+#
+#   To restart the container:
+#   ./restart.sh
+
+while true; do
+    rm -f restart.txt
+    
+    $* &
```

Comment:
FYI I am not very good at bash, and don't feel comfortable reviewing this part.

Reply:
lol it's very cryptic but also straightforward.

"$*" means "all the arguments"

"&" means "run this in the background"

so if I run
./start.sh sleep 1000
then this just runs
sleep 1000 &